app_flutter/lib/state/flutter_build.dart

```diff
@@ -51,6 +51,9 @@ class FlutterBuildState extends ChangeNotifier {
   List<String> _branches = <String>['master'];
   List<String> get branches => _branches;
 
+  final String _currentBranch = 'master';
+  String get currentBranch => _currentBranch;
```

Comment:
Is this used somewhere? I see `_currentBranch` is being used all the time.

Reply:
The followup PR to this will use this variable directly. I am going to leave this in to maintain consistency of the build state variables.

Reply:
SGTM.